File: models/reader.py
from dataclasses import dataclass
from datetime import datetime

@dataclass
class Reader:
    id: int | None = None
    account: str | None = None
    username: str | None = None
    password: str | None = None
    registration_duration: int | None = None # Assuming this maps to an int
    create_time: datetime | None = None
    update_time: datetime | None = None

File: src/main/java/org/xu/novel/domain/po/Reader.java
package org.xu.novel.domain.po;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Reader {
    private Integer id;
    private String account;
    private String username;
    private String password;
    private Integer registrationDuration;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
}
